src/api/app/models/project.rb

```diff
@@ -55,6 +55,8 @@ def autocomplete(search)
   has_many :linked_repositories, through: :path_elements, source: :link, foreign_key: :repository_id
   has_many :repository_architectures, -> { order('position') }, through: :repositories
 
+  has_many :watched_items, as: :watchable, dependent: :destroy
+  # FIXME: We will remove the following association when the one above goes out of beta
```

Comment:
To easily find this comment when we're removing the notifications_redesign feature toggle, I would include the feature toggle's name in the comment.

```suggestion
  # FIXME: We will remove the following association when the notifications_redesign goes out of beta
```

Reply:
But the feature toggle should be `new_watchlist` right?

Reply:
I added a comment mentioning the `new_watchlist` feature flag.

Reply:
> But the feature toggle should be `new_watchlist` right?

True! At the time of writing my previous comment, I didn't know we would have a new feature flag :slightly_smiling_face:.